mitodl/ocw-studio
# use OCW_COURSE_STARTER_SLUG in pipelines

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/1559

#### What's this PR do?
This PR uses the `OCW_COURSE_STARTER_SLUG` environment variable (added in https://github.com/mitodl/salt-ops/pull/1545) everywhere where `OCW_IMPORT_STARTER_SLUG` was being used. The variable was renamed to be more relevant to its use case. Not all references to it `OCW_IMPORT_STARTER_SLUG` are removed in this PR, it's still being set on pipelines for now until it is replaced in `ocw-hugo-themes` with the new variable

#### How should this be manually tested?
 - First of all, make sure you've taken care of the following prerequisites:
   - `ocw-studio` up and running
   - Minio support configured
   - Concourse support configured
   - `OCW_COURSE_STARTER_SLUG` set to `ocw-course-v2` in your `.env`
   - `OCW_HUGO_THEMES_BRANCH` set to `main` in your `.env`
   - `OCW_HUGO_PROJECTS_BRANCH` set to `main` in your `.env`
   - `fly` CLI installed locally and configured against your local Concourse target
 - Run `./manage.py backpopulate_pipelines --filter 9-40-introduction-to-neural-computation-spring-2018`
 - Run `fly -t local get-pipeline -p live/site:9-40-introduction-to-neural-computation-spring-2018 | grep OCW_COURSE_STARTER_SLUG` and ensure that you see `ocw-course-v2` being assigned, replacing the course name with one in your local database
 - Run `./manage.py upsert_mass_build_pipeline`
 - Run `fly -t local get-pipeline -p "mass-build-sites/offline:false,prefix:'',projects_branch:main,starter:'',themes_branch:main,version:live" | grep OCW_COURSE_STARTER_SLUG` and ensure that you see `ocw-course-v2` being assigned
